### ISSUE TITLE
Update restrictions to see a `chat_message`

### ISSUE THREAD

**Issue by GabrielInTheWorld** [OpenSlides/openslides-autoupdate-service#370]
Please update the [restrictions](https://github.com/OpenSlides/OpenSlides/wiki/chat_message-restrictions) to see a `chat_message`:

```
A user can see a chat_message if they have the permission chat.can_manage or they are in the respective chat_group/read_group_ids (given by the key chat_group_id).
```

**Comment by ostcar** [comment]
Should already be fixed with #369. Can you confirm, that you where using the current main branch with this commit?

**Comment by GabrielInTheWorld** [comment]
@ostcar I create this issue, because I updated the restrictions just a second before. I have not checked, if the restrictions are already up-to-date.

**Comment by ostcar** [comment]
Ahh I see. Sorry, I did not realize you changed the wiki page.

I don't see why your [change in chat_message](https://github.com/OpenSlides/OpenSlides/wiki/chat_message-restrictions/_compare/6c9f612b066e57b7d7736d573b05e27d0df5bc5c...3f534e466164cf0d6f187afbf0749ec55ffb32ac) is relevant.

If the user has the permission `chat.can_manage`, then he can see the respective `chat_group`. Is it ok for you that I revert your change in `chat_message`.

**Comment by GabrielInTheWorld** [comment]
> Ahh I see. Sorry, I did not realize you changed the wiki page.
> 
> I don't see why your [change in chat_message](https://github.com/OpenSlides/OpenSlides/wiki/chat_message-restrictions/_compare/6c9f612b066e57b7d7736d573b05e27d0df5bc5c...3f534e466164cf0d6f187afbf0749ec55ffb32ac) is relevant.
> 
> If the user has the permission `chat.can_manage`, then he can see the respective `chat_group`. Is it ok for you that I revert your change in `chat_message`.

Which change do you want to revert? 
Groups, which have "only" the permission to write to a chat_group, should not have indirectly also the permission to read every chat_message in the chat_group.

**Comment by ostcar** [comment]
Ok. Since you changed the wiki again, my point is not valid anymore.

I fixed this in #371
